lib/slim_fast/parser.ex

```diff
@@ -101,7 +101,7 @@ defmodule SlimFast.Parser do
              |> String.split(~r/^(-|==|=)/)
              |> List.last
              |> String.lstrip
-    inline = inline or String.starts_with?(input, "=")
+    inline = inline or String.starts_with?(input, "=") or String.starts_with?(script, "if")
```

Comment:
I think @Rakoth can contribute more than me here, since I'm not very familiar with the current implementation. But I'm happy to weigh in with my experiences of how Ruby Slim works as a daily user :)